tests/testthat/_snaps/snapshot.md

```diff
@@ -3,13 +3,6 @@
     Code
       snapshot()
     Output
-      The following package(s) are missing their DESCRIPTION files:
```

Comment:
This is a regression (or at least requires some changes to the test), because the point of these tests (at least as I understand them) is to ensure various problems are reported to the user.